include/ua_server.h

```diff
@@ -695,6 +699,20 @@ typedef struct {
                            void *sessionContext, const UA_NodeId *nodeId,
                            void *nodeContext, const UA_NumericRange *range,
                            const UA_DataValue *value);
+
+    /* Triggered, if the the node gets (un)monitored by a subscription.
+     *
+     * @param server Allows the access to the server object
+     * @param sessionId The session id, represented as an node id
+     * @param sessionContext An optional pointer to user-defined data for the specific data source
+     * @param nodeid Id of the node in question
+     * @param removed Determines, if the monitoring on the node was removed or created.
+     * @return Returns a status code for logging. It could be used to determine,
+     *         if the subscription was successfull and might be returned to the user. */
```

Comment:
The return value is not actually logged even if it indicates an error.
Either add the logging or make it a void return value.

Imho, an error code can also be returned with the notification values once the monitoreditems runs.
And removing a subscription should never fail. We cannot prevent clients from disconnecting.

Reply:
I added FIXMEs, where the callback returns. I was not sure, if it is okay to modify the result. 
If an hardware failure or something like this happens, it would be nice to be able to prevent the subscription in the first place.

You say the unsubscribe return value should not be modified, but it could be by `UA_Nodestore_getCopy()`. Do you consider this to be a valid change?